tests/helpers/reporter/metric/metric.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2016 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+//GetEnvironment detects the envieronment and return its struct
+func GetEnvironment() Environment {
+	environment := Environment{}
+	var osRelease = "/usr/lib/os-release"
+	var id, versionID string
+	if _, err := os.Stat(osRelease); err != nil {
+		osRelease = "/etc/os-release"
+	}
+
+	scanner, inFile := readFile(osRelease)
+	for scanner.Scan() {
+		info := strings.Split(scanner.Text(), "=")
+		switch value := info[0]; value {
+		case "ID":
+			id = info[1]
+		case "VERSION_ID":
+			versionID = info[1]
+		}
+	}
+	defer inFile.Close()
+
+	containerImage, _ :=
+		os.Readlink("/usr/share/clear-containers/clear-containers.img")
```

Comment:
to avoid hard coding path you can use ldflags  https://github.com/01org/cc-oci-runtime/blob/master/Makefile.am#L250